packages/office-ui-fabric-react/src/components/Rating/__snapshots__/Rating.test.tsx.snap

```diff
@@ -2,209 +2,251 @@
 
 exports[`Rating Renders Rating correctly 1`] = `
 <div
-  className="ms-Rating"
-  role="application"
+  aria-label=""
+  className="ms-Rating-star"
 >
   <div
+    aria-describedby={undefined}
     aria-labelledby={undefined}
-    className="ms-Rating-container"
-    role="radiogroup"
+    className="ms-FocusZone ms-Rating-focuszone"
+    data-focuszone-id="FocusZone2"
+    onFocus={[Function]}
+    onKeyDown={[Function]}
+    onMouseDownCapture={[Function]}
+    role="presentation"
```

Comment:
Am really confused by role=presentation AND tabIndex = -1. Why are either or these needed? why is data-is-current needed?